eve/workers/pod-unit-tests/pod.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "scality-core-ui-unit-tests-worker"
+spec:
+  containers:
+  - name: worker
```

Comment:
use the `env` key for the `CI` environment variable here, so you don't need to change `package.json`.